tests/test_dislocation.py

```diff
@@ -731,6 +731,17 @@ def test_cylinder_ovito_dxa(self, disloc, gen_bulk,
         # err = angle - ref_angle
         # print(f'angle = {angle} ref_angle = {ref_angle} err = {err}')
         # assert abs(err) < tol
+
+    def test_displacement_r_sc(self, disloc, subtests):
```

Comment:
better test would have shown the previously reported issue with a vacancy

Reply:
Have changed this test - only for BCCEdge100Dislocation, and it looks to see if the dislocation generated using r_sc has more bulk-coordinated atoms (1st nearest neighbour) than the full SC disloc - should catch the vacancy issue, and we can easily expand it to handle additional dislocation types quite easily.